sycl/source/detail/device_image_impl.hpp

```diff
@@ -47,7 +47,7 @@ class device_image_impl {
 
     // Collect kernel names for the image
     pi_device_binary DevBin =
-        const_cast<pi_device_binary>(&BinImage->getRawData());
+        const_cast<pi_device_binary>(&MBinImage->getRawData());
```

Comment:
Do we really need MBinImage field as soon as it is used only once in constructor?

Reply:
Yes, the implementation is not complete yet, it will be used more later.